alacritty/src/event.rs

```diff
@@ -716,6 +712,8 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
                 self.terminal.vi_goto_point(*hint_bounds.start());
             },
         }
+
+        self.mark_dirty();
```

Comment:
Which action here requires being marked dirty? We shouldn't just randomly sprinkle this around because it seems convenient. Most of the actions taken here don't seem to require any damage?

The only things I could imagine would be `MoveViModeCursor` and `Select`?

Reply:
Yes, due to Vi movements it's here.

Reply:
Should only be in the vi branches then probably?